### DISCUSSION TITLE
About Context Menu & Customized explorer

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#725] [Ideas]
I'm using directory opus (it is like total commander thing. customize explorer). yesterday I try to set default for using 'open containing folder' in context menu. (every time it open file explorer.) But it seemed that there was no option in FL.

Today I found solution for directory opus, just insert directory opus "full path" in custom explorer in setting.
everything and program has this option but Explorer plugin doesn't. It looks necessary.
and it need a little guide everything and program section, too.... I didn't even think about it.
@taooceros said 'I think it is reasonable to have a global setting for that accessed via ipublicapi'.
yeah, it need to change to global setting.

And Each plugin has a different context. It is weird.
How about use https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu like that?
I think it's better solution. Listary uses this method. I recommend you try it on.
https://www.listary.com/download/beta/listary6/ListaryInstaller.exe
https://discussion.listary.com/t/listary-6-beta/4615

There was also this request in wox, 
https://github.com/Wox-launcher/Wox/issues/107
funny thing is that @jjw24 merged this issue.

**Comment by taooceros** [comment]
> And Each plugin has a different context. It is weird.

Actually, I would consider this is expected because this allows plugins to develop their own feature set.

> How about use https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu like that?

That's interesting, and probably we will be able to use it for Explorer or everything plugin in the future.

**Reply by onesounds** [reply]
yes. some plugin need original context menu. but I think it can be add, not change.
